src/Executables/ExportEquationOfStateForRotNS/ExportEquationOfStateForRotNS.cpp

```diff
@@ -0,0 +1,190 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <boost/program_options.hpp>
+#include <cmath>
+#include <cstddef>
+#include <fstream>
+#include <iomanip>
+#include <iterator>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "Options/Options.hpp"
+#include "Options/ParseOptions.hpp"
+#include "Parallel/Printf.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "PointwiseFunctions/Hydro/Units.hpp"
+#include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/FileSystem.hpp"
+#include "Utilities/TMPL.hpp"
+
+// Charm looks for this function but since we build without a main function or
```

Comment:
There is a main function here. The comment probably still applies because we link in Charm but don't use it?

Reply:
Yes, this is a charm function we need to define. I've added a bit of wording that hopefully makes this clearer.